_posts/2017-12-19-fki0004.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+category: podcast
+title:  "FKI0004 Not Acceptable
+date:   2017-12-19 22:00:00+01:00
+file_url: "https://s3.eu-central-1.amazonaws.com/feinkost-intenet/fki0004.mp3"
```

Comment:
Auch hier erstmal auf s3 gehostet, da cast die Datei verweigert, da die länge nicht mit der Aufnahme übereinstimmt. (Wurde aus mehreren parts zusammen geschnitten)

Problem wie auch in #19 das ich noch nicht weiß wie wir da download Statistiken bekommen.

Reply:
Ich würde mal Blubrry ausprobieren #21